pydantic/json_schema.py

```diff
@@ -222,16 +222,15 @@ def generate_definitions(
         returns the generated definitions paired with a mapping from the input keys to the definition references.
 
         Args:
-            inputs (Sequence[tuple[JsonSchemaKeyT, JsonSchemaMode, core_schema.CoreSchema]]): A sequence of tuples,
-                where:
+            inputs: A sequence of tuples, where:
```

Comment:
I took the opportunity to remove the types on the docstrings. I can add them back if you think they should be done separately.